pkg/staticnetworkconfig/generator.go

```
@@ -112,6 +112,8 @@ func (s *StaticNetworkConfigGenerator) executeNMStatectl(ctx context.Context, ho
 		s.log.WithError(err).Errorf("Failed to write host config to temp file")
 		return "", err
 	}
+	_ = f.Sync()
+	_ = f.Close()
```

Why ignore the errors? At least log a warning or something.